app/views/posts/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class='single-user-info'>
+    <h3>Post #<%= index + 1 %></h3>
+    <p><%= post.text%>...</p>
+    <div class='counters'>
+    <span>Comments: <%= post.comments.size ||= 'No comments yet' %></span>,
```

Comment:
- [ ] [REQUIRED] Kindly use the recent_comments method to to render 5 most recent comments

```ruby
<% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>
```